src/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import './App.css';
+import './styles/output.css';
+import NavBar from './components/NavBar';
+import Manager from './components/Manager';
+import Footer from './components/Footer';
+function App() {
+  const [errors, setErrors] = useState({});
+
+  return (
+    <div className="bg-blue-100 min-h-screen flex flex-col">
+      <NavBar />
+      <Manager setErrors={setErrors} />
```

Comment:
how about renaming `Manager` to something like `CredentialForm`?

Here, this is just a suggestion, feel free to provide some other name if you like.